capa-controller-role/setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -u
+
+BLUE='\033[0;34m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
```

Comment:
I decide to use an script instead of old copy&paste since I find it easier to read and mantain